maven-surefire-plugin/src/site/apt/examples/toolchains.apt.vm

```diff
@@ -54,3 +54,36 @@ Using Maven Toolchains with ${thisPlugin}.
 +---+
 
          The above example assumes that your toolchains.xml contains a valid entry with these values.
+
+* Minimal Java Versoin for Toolchains
+
+    Unfortunately, the minimum version of Java that can be used as a toolchain is limited by the Surefire Booter requirement.
+
+*---------------------------------------------+---------------------------+-------------------------+
+|| <<Surefire Version>>                       || <<Minimal Java Version>> || <<Class File Version>> |
+*---------------------------------------------+---------------------------+-------------------------+
+| 3.0.0-M6 or higher                          |    Java 8                 |    52                   |     
+*---------------------------------------------+---------------------------+-------------------------+
+| 3.0.0-M5 and lower                          |    Java 7                 |    51                   |  
+*---------------------------------------------+---------------------------+-------------------------+
+| To Be Completed                             |    Java 6                 |    50                   |    
```

Comment:
Here it is d23e970cd86ce4896edf5f8ad1f1acd66a9a27c2, https://issues.apache.org/jira/browse/SUREFIRE-1474, so everything before 3.0.0-M1

Reply:
I think everything before 6 is irrelevant these days... no need to put too much time into it.

Reply:
Ok so I just added missing information about java 6